native/android/app/src/main/java/com/notes/fxaclient/FxaClientModule.java

```diff
@@ -96,19 +112,55 @@ public void begin(final Callback successCallback,
         });
     }
 
+    @ReactMethod
+    public void renewToken(String state, final Callback successCallback,
+                           final Callback errorCallback) {
+        FxaResult<FirefoxAccount> account;
+        if (this.account != null) {
+            account = FxaResult.Companion.fromValue(this.account);
+        } else if (state != null) {
+            Log.e("called renew", "help");
```

Comment:
We can do proper logging here if we want to . Maybe added proper logging to other branches as well?